eslint.config.mjs

```diff
@@ -1,11 +1,12 @@
 import globals from 'globals';
 import pluginJs from '@eslint/js';
 import prettierPlugin from 'eslint-plugin-prettier';
```

Comment:
This one have some weird errors, maybe it is not compatible with our EsLint version?